test/postinstall/README.md

```diff
@@ -0,0 +1,41 @@
+# Post-install tests
+
+The purpose of these tests is to check if an installed PROJ library can be used to configure and build other applications. It provides additional test coverage for the CMake config files and with `proj.pc` used for pkg-config.
+
+## Applications
+
+The two trivial applications `c_app` and `cpp_app` use C and C++ compilers, respectively. They function similarly, taking only one flag `-t` for an example coordinate transform, `-s` to show searchpath, and `-v` for the version of PROJ.
+
+## Configure method
+
+Each configure method is driven using a script with two arguments:
+1. Path to a PROJ install prefix (containing `bin`, `lib`, etc.)
+2. An optional build mode `shared` (default) or `static`
+
+### CMake
+
+Use `test_cmake.sh` for Unix-like or `test_cmake.bat` for Windows. This checks that `find_package(PROJ)` (or legacy `find_package(PROJ4)`) can be used with `target_link_libraries(mytarget PRIVATE PROJ::proj)`. The C-based app enables C++ when linking to static libproj. These scripts will only work with PROJ built using CMake, and will not work with PROJ built using Autotools.
+
+### Makefile / pkg-config
```

Comment:
What does this configuration brings compared to autotools ?

Reply:
A few reasons: it's simpler to see what's going on, and can be run in test environments without automake/conf. The test scripts are shared between the two, so there isn't too much code overlap.

I'll check again to see if these two tests yields different results with macOS. There were are few gotchas with rpaths, which I'll explain in more detail later.

Reply:
My feeling is that automake/conf should be sufficient. I'm not sure what environments do not have autotools that we'd want to support. The main thing we want to check is that the paths in pkg-config are OK, whatever hand-made makefile vs autotools consume it, don't we ?

Reply:
Sure, there's no burden to drop these tests. I think all of the test environments currently have automake/conf.

Reply:
Oh right, it was MSYS2 (in windows.yml) that only had make. Perhaps keep this test script, but don't duplicate them where autotools is available? I'll tack on a second commit to show this. The alternative is to install packages for MSYS2.